data/transactions/logic/evalStateful_test.go

```diff
@@ -2925,54 +2469,10 @@ func TestAppLoop(t *testing.T) {
 	// Double until > 10. Should be 16
 	testApp(t, stateful+"int 1; loop: int 2; *; dup; int 10; <; bnz loop; int 16; ==", ep)
 
-	testApp(t, stateful+"int 1; loop: int 2; *; dup; int 10; <; bnz loop; int 16; ==", ep)
-
 	// Infinite loop because multiply by one instead of two
 	testApp(t, stateful+"int 1; loop:; int 1; *; dup; int 10; <; bnz loop; int 16; ==", ep, "dynamic cost")
 }
 
-func TestWriteLogs(t *testing.T) {
```

Comment:
None of that happens any more.  Instead of jamming the Logs down into the ledger and then yanking them back out to make the EvalDeltas, they are pulled directly out of the EvalContext to make the EvalDelta.